### DISCUSSION TITLE
next_auth_providers_credentials__WEBPACK_IMPORTED_MODULE_1__ is not a function

### DISCUSSION THREAD

**Discussion by Iulian-Dragomirescu** [nextauthjs/next-auth#7147] [Help] [answered]
### Question 💬

i try to use next-auth. I did step by step what it says in the documentation and i have this error

### How to reproduce ☕️

![image](https://user-images.githubusercontent.com/54176220/229586382-cfffbf62-c565-4351-baf2-78761380ddc1.png)

// [...nextauth].js
![image](https://user-images.githubusercontent.com/54176220/229586464-cc37ce65-6c4b-4fe0-8773-d24326e833fe.png)
![image](https://user-images.githubusercontent.com/54176220/229586755-f42e14bd-41c4-4b94-8e55-368ca6d35443.png)

I really don't know what I'm doing wrong. can you help me?

    "next": "13.2.4",
    "next-auth": "^4.21.1",

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by GabrielCartier** [comment, marked as answer]
I was having similar issues when I was importing my `authOptions` from a package from the workspace when that package was defined as `type: "module"`. 

Not sure that's your case here but I'm interested in seeing the answers for this.

**Reply by ntzolov** [reply]
10x a lot for that comment! I've been searching for a solution to that 2 days! 😒

**Comment by Iulian-Dragomirescu** [comment]
pff, i have ```type: "module"``` , but all my node backend use imports.. can i do something but without removing all imports from node?

**Comment by GabrielCartier** [comment]
try removing the `type: "module"` to see if it works. Not sure why exactly the import is not working properly for modules. Still investigating the bug.

**Comment by Iulian-Dragomirescu** [comment]
ok should i report that there is a bug?

**Comment by GabrielCartier** [comment]
Shouldn't necessarily be a discussion IMO, is there a reason why it doesn't work with `type:module`?

My solution was to extract my `auth` options in a separate project and continue using `type:module` in the rest of the API project.
